### PR TITLE
Add redirectPermanentlyTo and redirectTemporarilyTo functions

### PR DESCRIPTION
**Pull request by dpraimeyuu**
This small PR makes `redirect` capability more verbose by getting rid of additional parameters required from the caller.

Before:
```fsharp
Response.redirect "/" true
```

After:
```fsharp
Response.redirectPermanentlyTo "/"
```

### CODE REVIEW

File: test/Falco.Tests/Falco.Tests.fsproj [hunk 1]
```diff
@@ -20,6 +20,7 @@
     <Compile Include="AuthTests.fs" />
     <Compile Include="RequestTests.fs" />
     <Compile Include="RequestHandlersTests.fs" />
+    <Compile Include="Extensions/ResponseTests.Extensions.fs" />
```
Comment (pimbrouwers) [inline test/Falco.Tests/Falco.Tests.fsproj:23]: I would rather we didn't add this file, and stick with the inline Recevied call.

Comment (dpraimeyuu) [reply test/Falco.Tests/Falco.Tests.fsproj:23]: I didn't want to expose details what does it mean that we have temporary/permanent redirect :+1: I will change it and leave it inlined!

Comment (dpraimeyuu) [reply test/Falco.Tests/Falco.Tests.fsproj]: Done! :rocket:

File: src/Falco/Response.fs [hunk 2]
```diff
@@ -86,12 +86,31 @@ let private writeString (encoding : Encoding) (httpBodyStr : string) (ctx : Http
         let httpBodyBytes = encoding.GetBytes httpBodyStr
         writeBytes httpBodyBytes ctx
 
+[<Obsolete("Response.redirect is deprecated, please use Request.redirectPermanentlyTo or Request.redirectTemporarilyTo respectively")>]
```
Comment (pimbrouwers) [inline src/Falco/Response.fs:89]: Can you remove this? The plan is to add these to the master branch, do a final patch version with all the obsolete tags and new funcs.

Comment (dpraimeyuu) [reply src/Falco/Response.fs:89]: Of course!

**Review by pimbrouwers — CHANGES_REQUESTED** [review]

**Comment by dpraimeyuu** [comment]
Closing this one to keep it without merging `master` to `develop` changes. Implemented all your suggestions, please check #80

**Pull request closed without merging**